.eslintrc.js

```diff
@@ -1678,8 +1678,6 @@ module.exports = {
       files: ['**/*'],
       rules: {
         ...require('eslint-config-prettier').rules,
-        ...require('eslint-config-prettier/react').rules,
-        ...require('eslint-config-prettier/@typescript-eslint').rules,
```

Comment:
These two rule-sets are now included in the base rule-set, so no need to manually require them anymore